CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+### 0.10.0 (unreleased)
+
+- Breaking changes
+  - Auto derivation of readers and writers for case classes and sealed traits is now disabled by default. Now users need
+    to import `pureconfig.generic.auto._` everywhere a config is loaded or written (e.g. in files with calls to
+    `loadConfig`);
+  - The `AllowMissingKeys` trait was renamed to `ReadsMissingKeys`;
+  - Dropped support for Scala 2.10.
+
+- New features
+  - The auto-derivation features of PureConfig, powered by shapeless, were extracted to a separate `pureconfig-generic`
+    module, while `pureconfig-core` was left with only the absolute minimum for PureConfig to be useful. `pureconfig`
+    will continue to be published as a Maven artifact aggregating the two aforementioned artifacts;
+  - Users have now more control over reader and writer derivation. See the
+    [docs](https://pureconfig.github.io/docs) for more information;
```

Comment:
I'll update this link to point to the correct page later. If I do it now, the build will fail since the `non-automatic-derivation.html` page doesn't exist yet.